pages/Documents/MessagingChannels/ConnectToMessaging/api.md

```diff
@@ -0,0 +1,254 @@
+---
+pagename: API
+redirect_from:
+  - ConnectToMessaging.html
+Keywords:
+sitesection: Documents
+categoryname: "Messaging Channels"
+documentname: Connect To Messaging
+permalink: connect-to-messaging-api.html
+indicator: messaging
+---
+
+### Introduction
+
+Connect To Messaging (C2M) is a product offering from LivePerson allowing brands to offer consumers an option to deflect to messaging when they call into their IVR. C2M API serves as an intermediary between the brand’s IVR System and LivePerson Conversational Cloud, ensuring that the consumer is invited to join a conversation with an agent via eligible messaging channels. Once a consumer responds to a message from that channel, C2M ensures that the conversation is routed to an agent of the appropriate skill specified by the brands.
+
+
+### Getting Started
+
+1. Onboarding to C2M is mandatory before it.  
```

Comment:
do we have a link for the onboarding doc?

Reply:
It's under review and will add the link then.